packages/webdriver/src/command.ts

```diff
@@ -99,6 +99,10 @@ export default function (
             }
 
             this.emit('result', { method, endpoint, body, result })
+
+            if (commandUsage.includes('deleteSession')) {
```

Comment:
Let's be more strict here.

```suggestion
            if (command === 'deleteSession') {
```